pkg/broker/types.go

```diff
@@ -47,13 +49,14 @@ type ServiceInstance struct {
 // CreateServiceInstanceRequest represents a request to a broker to provision an
 // instance of a service
 type CreateServiceInstanceRequest struct {
-	OrgID             string                 `json:"organization_guid,omitempty"`
-	PlanID            string                 `json:"plan_id,omitempty"`
-	ServiceID         string                 `json:"service_id,omitempty"`
-	SpaceID           string                 `json:"space_guid,omitempty"`
-	Parameters        map[string]interface{} `json:"parameters,omitempty"`
-	AcceptsIncomplete bool                   `json:"accepts_incomplete,omitempty"`
-	ContextProfile    ContextProfile         `json:"context,omitempty"`
+	OrgID               string                 `json:"organization_guid,omitempty"`
+	PlanID              string                 `json:"plan_id,omitempty"`
+	ServiceID           string                 `json:"service_id,omitempty"`
+	SpaceID             string                 `json:"space_guid,omitempty"`
+	Parameters          map[string]interface{} `json:"parameters,omitempty"`
+	AcceptsIncomplete   bool                   `json:"accepts_incomplete,omitempty"`
+	ContextProfile      ContextProfile         `json:"context,omitempty"`
+	OriginatingUserInfo v1.UserInfo            `json:"user,omitempty"`
```

Comment:
Went back and forth as to whether this should be added here, but it seems to make the most sense since we require it for the creation of a service instance.

Reply:
What about putting it in the ```ContextProfile``` that has things like the platform and the namespace